software/verification/verify_cdl_sequence.py

```diff
@@ -102,15 +104,45 @@ def parse_configurations(self):
 
             self.test_list.append(reference)
 
+    def run_modelica_json(self, model, model_dir):
+        """this function uses the modelica-json tool to convert modelica files to json
+
+        Parameters:
+        -----------
+        model : str
+                name of modelica model to be translated to json
+        model_dir : str
+                    directory where to store the json output
+        """
+        app_js_path = os.path.join(self.modelica_json_path, 'app.js')
+        model_path = model.replace('.', '/') + '.mo'
```

Comment:
Use `os.path.sep` everywhere (and `os.path.join()` on line 123 to make sure it is portable.